src/bindings/python/src/pyopenvino/graph/node_input.cpp

```diff
@@ -86,6 +87,21 @@ void regclass_graph_Input(py::module m) {
                 :return: A dictionary of user defined data.
                 :rtype: openvino.RTMap
              )");
+    input.def(
+        "set_rt_info",
+        [](ov::Input<ov::Node>& self, const py::object& obj, const py::str& path) -> void {
+            self.get_rt_info()[path.cast<std::string>()] = Common::utils::py_object_to_any(obj);
+        },
+        py::arg("obj"),
+        py::arg("path"),
```

Comment:
Shouldn't it be named 'key'?

Reply:
In most `rt_info` methods in python api it was _path_. So I sticked to it.
https://github.com/openvinotoolkit/openvino/blob/bf846831916a363ac86b79080fde33cfaf9f14d8/src/bindings/python/src/pyopenvino/graph/model.cpp#L1430-L1431
I believe it is because `Model.set_rt_info` can also take as input a list of strings which defines a path.
I can update all `set_rt_info`  methods in python api that take single string as path and name this argument 'key'